eclass/mozcoreconf-v6.eclass

```diff
@@ -207,18 +207,19 @@ mozconfig_init() {
 	use custom-cflags || strip-flags
 
 	# Additional ARCH support
+
+	if use arm; then
+		if tc-ld-is-bfd ; then
```

Comment:
I am not sure if this will work when USE=clang is set because USE=clang will force lld linker via configure.

Reply:
Well, the combination of CC=clang and use of ld.bfd works for arm, but that doesn't mean USE=clang will work with ld.bfd. But I think I made a fundamental flaw anyway: there is no `tc-ld-is-bfd` from the toolchain-funcs eclass, right?

Reply:
Yup, toolchain-funcs can only detect gold.

Reply:
How do you force *bfd* and *gold* linker usage when *USE=clang* is set?

I don't think toolchain-funcs can help here because Mozilla's build system is ignoring *LD* value (see message in build.log) and instead we set linker via configure.

Reply:
I didn't. I shuffled through the eclass and saw that firefox will fail to link on arm, unless ld.bfd is forced to use or a guard will be put up against those cmds incompatible to ld.gold and/or ld.ldd 

I'm going to test gold and ldd with clang now, maybe we're lucky and ldd is working just fine on arm.